### ISSUE TITLE
Use webL10n on documentFragment

### ISSUE THREAD

**Issue by Phyks** [fabi1cazenave/webL10n#73]
Hi,

The function 
```javascript
  function translateFragment(element) {
    element = element || document.documentElement;

    // check all translatable children (= w/ a `data-l10n-id' attribute)
    var children = getTranslatableChildren(element);
    var elementCount = children.length;
    for (var i = 0; i < elementCount; i++) {
      translateElement(children[i]);
    }

    // translate element itself if necessary
    translateElement(element);
  }
```

in webL10n should be able to translate a documentFragment, I think, and is exposed as `translate` in the public API.

But the last call to `translateElement(element)` fails in such a case as it is not possible to `getAttributes` of a documentFragment. What about encapsulating it in a test to not call it on documentFragment elements?

**Comment by Rob--W** [comment]
Patches are welcome. Best fix is probably checking for `nodeType === 1` in translateElement (Node.ELEMENT_NODE) and returning otherwise.

**Comment by Phyks** [comment]
Closed by PR #74.
